pkg/features/features.go

```diff
@@ -76,6 +76,12 @@ const (
 	// owner: @dimityrmirchev
 	// alpha: v1.93.0
 	ShootManagedIssuer featuregate.Feature = "ShootManagedIssuer"
+
+	// BilinearPodAutoscalingForAPIServer is applied to a seed cluster and enables simultaneous independent horizontal and vertical scaling.
+	// This feature is incompatible with the HVPA and HVPAForShootedSeed features.
```

Comment:
+1 for using the same text as in `feature_gates.md`:

```suggestion
	// BilinearPodAutoscalingForAPIServer enables a new autoscaling mechanism for the shoot kube-apiserver 
	// where it is scaled simultaneously via HPA on request rate and VPA on resource usage.
	// When enabled, this feature takes prececence over the HVPA and HVPAForShootedSeed features
	// in determining how shoot kube-apiserver pods are scaled.
```